docs/vagrant.md

```diff
@@ -1,35 +1,52 @@
 The **LibreTime Vagrant install** is the fastet way to get LibreTime up and running in a way
-to hack on its source code or to test it locally.
+to hack on its source code or to test it locally. There are two supported providers: libvirt
+and VirtualBox. Libvirt provider is supported out of the box under Linux, while VirtualBox is
```

Comment:
I think this depends on the OS you're running on, at least on my Fedora machine the packagers don't stray from the default of using VirtualBox so this is probably specific to .deb packaging.

Reply:
Fixed in the new commit